File: baekjoon/java/src/bronze3/Q2588_Multiply.java
package bronze3;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

// https://www.acmicpc.net/problem/2588
public class Q2588_Multiply {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int input1 = Integer.parseInt(br.readLine());
        int input2 = Integer.parseInt(br.readLine());

        System.out.println(input2 % 10 * input1);
        System.out.println(input2 / 10 % 10 * input1);
        System.out.println(input2 / 100 * input1);
        System.out.println(input2 * input1);
    }
}

File: baekjoon/kotlin/src/main/kotlin/bronze3/Q2588_Multiply.kt
package botech.bronze3

import java.io.BufferedReader
import java.io.InputStreamReader

class Multiply

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val inputA = br.readLine()
    val inputB = br.readLine()

    val intA = inputA.toInt()
    val intB = inputB.toInt()

    println(intB % 10 * intA)
    println(intB % 100 / 10 * intA)
    println(intB / 100 * intA)
    println(intA * intB)
}
